linera-rpc/src/message.rs

```diff
@@ -190,6 +196,18 @@ impl TryFrom<RpcMessage> for Vec<Certificate> {
     }
 }
 
+impl TryFrom<RpcMessage> for Vec<CryptoHash> {
+    type Error = NodeError;
+    fn try_from(message: RpcMessage) -> Result<Self, Self::Error> {
+        use RpcMessage::*;
```

Comment:
I'd prefer to not import enum variants.
(As @ndr-ds pointed out recently, that can be dangerous for ones without fields. But even for the ones with fields, I'd say it's safer to fully qualify them, so we're safe even if the fields are removed later.)

Reply:
So, we would like to have a type
```rust
struct CryptoHashes {
  hashes: Vec<CryptoHash>,
}
```
as well as similar for `Vec<BlobId>`.

So, that we so not have the implementation of `From/TryFrom` for `Vec<_>`?

Reply:
Not sure; that's not what I meant. I'd just remove the `use RpcMessage::*` and fully qualify the enum variant below.

Reply:
Ok, thank you. It is good that I asked about that problem.